app/templates/components/nav-bar.hbs

```diff
@@ -46,7 +46,7 @@
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
         <div class="item">
```

Comment:
Instead of this, make the whole item a link and add the `ui blue` class here so that it is styled correctly.